scripts/enable_hourly_updates.py

```
@@ -179,17 +182,32 @@ def main():
         print("✅ Found GitHub token!")
+        # One-time migration: if we read from legacy path, ensure new path has the token too
+        new_token_path = Path.home() / '.gamecache' / 'token.json'
+        if token_file != new_token_path:
+            try:
+                new_token_path.parent.mkdir(exist_ok=True)
+                with open(new_token_path, 'w') as nf:
+                    json.dump(token_data, nf, indent=2)
+                try:
+                    os.chmod(new_token_path, 0o600)
+                except Exception:
+                    pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)